src/versions.ts

```diff
@@ -1,2 +0,0 @@
-export const DOTLOTTIE_PLAYER_VERSION = "1.3.1"; 
-export const LOTTIE_WEB_VERSION = "^5.10.0";
```

Comment:
wouldn't it introduce breaking changes to products using it ?

Reply:
No it wont because the getVersions() method is used, users dont get the version from this file